dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/MockedBeamSqlTable.java

```diff
@@ -49,6 +48,7 @@
   public static final ConcurrentLinkedQueue<BeamSqlRow> CONTENT = new ConcurrentLinkedQueue<>();
 
   private List<BeamSqlRow> inputRecords;
+  private PCollection.IsBounded isBounded = PCollection.IsBounded.BOUNDED;
```

Comment:
@XuMingmin make sense. Let me add a `MockedUnboundedTable` and add real test for unbounded table.